sample/src/main/java/io/ona/kujaku/sample/activities/MainActivity.java

```diff
@@ -490,15 +488,27 @@ public String readAssetFile(String inFile) {
 
     private void requestBasicPermissions() {
         ArrayList<String> notGivenPermissions = new ArrayList<>();
+        String[] basicPermissions;
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) { // Android 13 (API 33)
+            basicPermissions = new String[]{
+                    Manifest.permission.MANAGE_EXTERNAL_STORAGE
+            };
+        } else {
+            basicPermissions = new String[]{
+                    Manifest.permission.READ_EXTERNAL_STORAGE,
+                    Manifest.permission.WRITE_EXTERNAL_STORAGE
+            };
+        }
 
         for (String permission : basicPermissions) {
             if (!Permissions.check(this, permission)) {
                 notGivenPermissions.add(permission);
             }
         }
 
-        if (notGivenPermissions.size() > 0) {
-            Permissions.request(this, notGivenPermissions.toArray(new String[notGivenPermissions.size()]), PERMISSIONS_REQUEST_CODE);
+        if (!notGivenPermissions.isEmpty()) {
```

Comment:
```suggestion
        if (notGivenPermissions.isNotEmpty()) {
```

Reply:
@ellykits 
isNotEmpty will not be resolved for ArrayList with this java version